src/dune_rules/lib_mode.mli

```diff
@@ -0,0 +1,55 @@
+type t =
+  | Ocaml of Ocaml.Mode.t
+  | Melange
+
+module Cm_kind : sig
+  type t =
+    | Ocaml of Ocaml.Cm_kind.t
+    | Melange of Melange.Cm_kind.t
+
+  val source : t -> Ocaml.Ml_kind.t
+
+  val ext : t -> string
+
+  val cmi : t -> t
+
+  val to_dyn : t -> Dyn.t
+
+  module Map : sig
+    type cm_kind = t
```

Comment:
Make this an alias please

Reply:
done in https://github.com/ocaml/dune/pull/6238/commits/07c6f7095af189fd3fa08194c055b090def6b361.